src/main/java/org/spongepowered/api/event/network/ClientConnectionEvent.java

```diff
@@ -160,7 +179,23 @@
          * @param world The world
          * @param transform The transform
          */
-        void setLoginPosition(World world, Transform transform);
+        void setLocation(World world, Transform transform);
+
+        /**
+         * Sets the {@link Location} the {@link User} will have
+         * at login.
+         *
+         * @param location The location
+         */
+        void setLocation(Location location);
+
+        /**
+         * Sets the rotation the {@link User} will have
+         * at login.
+         *
+         * @param rotation The rotation
+         */
+        void setRotation(Vector3d rotation);
```

Comment:
For the other events like this, I didn't separate out rotation. It is easy enough to construct a transform to set it. I don't want to start adding all the methods to set the stuff on transform.